WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoViewModel.swift

```diff
@@ -12,8 +12,11 @@ final class ProductCreationAIStartingInfoViewModel: ObservableObject {
 
     @Published private(set) var imageState: ImageState
     @Published var isShowingMediaPickerSourceSheet: Bool = false
+    @Published var isShowingViewPhotoSheet: Bool = false
```

Comment:
Super nit:

```suggestion
    @Published var isShowingMediaPickerSourceSheet = false
    @Published var isShowingViewPhotoSheet = false
```

Reply:
Update in https://github.com/woocommerce/woocommerce-ios/pull/13213/commits/9e98a5b54023ff12acb98d103113c5cc8ec96fd7